File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/security/JwtCookieHelper.java
package uk.gov.justice.digital.hmpps.oauth2server.security;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;
import uk.gov.justice.digital.hmpps.oauth2server.utils.CookieHelper;

@Component
public class JwtCookieHelper extends CookieHelper {
    @Autowired
    public JwtCookieHelper(final JwtCookieConfigurationProperties properties) {
        super(properties.getName(), properties.getExpiryTime(), properties.isSecure());
    }
}

File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/security/JwtCookieHelper.kt
package uk.gov.justice.digital.hmpps.oauth2server.security

import org.springframework.stereotype.Component
import uk.gov.justice.digital.hmpps.oauth2server.utils.CookieHelper

@Component
class JwtCookieHelper(properties: JwtCookieConfigurationProperties) : CookieHelper(properties.name, properties.expiryTime)
